linkml_runtime/loaders/json_loader.py

```diff
@@ -25,3 +25,28 @@ def loader(data: Union[str, dict], _: FileInfo) -> Optional[Dict]:
             metadata.base_path = base_dir
         return self.load_source(source, loader, target_class,
                                 accept_header="application/ld+json, application/json, text/json", metadata=metadata)
+
+@click.group()
+def cli():
+    pass
+@cli.command('json_load')
+@click.option('-i', '--input', help='Data source that needs to be loaded.')
+@click.option('-t', '--target', help=' Target class')
+@click.option('-b', '--base-dir', help='Base directory (optional)')
+@click.option('-m', '--metadata', help='Metadata (optional)')
+
+
+def load_cli(input:str, target:str, base_dir:str, metadata:str) -> YAMLRoot:
+    """Loads data from source \n
+    Args: \n
+        input (str): Input data to load \n
+        target (str): Target Class \n
+        base_dir (str): Base directory \n
+        metadata (str): Metadata \n
+    Returns: \n
+        YAMLRoot
+    """
+    JSONLoader.load(source=input, target_class=target, base_dir=base_dir, metadata=metadata)
```

Comment:
have you tried this?

The signature of load is

```python
   def load(self, source: Union[str, dict, TextIO], target_class: Type[YAMLRoot], *, base_dir: Optional[str] = None,
 ```

but your target_class is a string - does it get auto-converted?

Reply:
No it does not .Fixed it.